src/pg_query_deparse.c

```diff
@@ -10,6 +10,81 @@
 
 #include "protobuf/pg_query.pb-c.h"
 
+PgQueryDeparseResult pg_query_deparse(Node *node)
```

Comment:
I think it'd be better to leave the decision whether its a node (that potentially needs to be wrapped) or a list of RawStmt nodes to the caller, so if we introduce support for deparsing expressions (instead of raw statements) in the future, we don't have a generic function name making the API confusing to understand.